fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -229,12 +229,19 @@ class DefaultBrowserToolbarController(
         internal const val TELEMETRY_BROWSER_IDENTIFIER = "browserMenu"
     }
 
+    /**
+     * Stop showing the CFR after being displayed three times with
+     * with at least 12 hrs in-between.
+     * As described in: https://bugzilla.mozilla.org/show_bug.cgi?id=1861173#c0
+     */
     private fun updateShoppingCfrSettings() = with(activity.settings()) {
-        if (reviewQualityCheckCfrDisplayTimeInMillis != 0L) {
-            // We want to show the first CFR a second time if the user doesn't opt in the feature
+        if (reviewQualityCheckCfrDisplayTimeInMillis != 0L &&
+            reviewQualityCheckCFRShownCounter >= 2
```

Comment:
Shouldn't this be 3? 
Let's make this a constant with a name that acts as a documentation that we want to display this CFR 3 times.